operator/v1/types_ingress.go

```diff
@@ -225,6 +241,65 @@ var (
 	DNSReadyIngressConditionType = "DNSReady"
 )
 
+// Ciphers defines the cipher algorithms used by an IngressController for performing
+// encryption or decryption of network connections. A profile can be specified or a
+// custom strategy of ciphers.
+type Ciphers struct {
+	// Profile is one of three recommended cipher configurations from [1].
+	//
+	// [1] https://wiki.mozilla.org/Security/Server_Side_TLS#Recommended_configurations
+	//
+	// +optional
+	Profile CipherProfile `json:"profile,omitempty"`
+	// CustomStrategy consists of a manually defined list of ciphers with each
+	// cipher separated by a ":".
```

Comment:
This phrasing might be clearer: "an explicit, colon-delimited list of ciphers".

Reply:
Thanks for the recommendation. I have updated the phrasing.